projects/client-certificate-authentication/keycloak/README.md

```diff
@@ -71,7 +47,7 @@ The instructions here are for macOS only, but dnsmasq works on Linux as well.
   * If you are using an Android emulator running on the same machine, you don't need to do anything if the host machine's DNS server entry is updated.
   * If you are using an Android device, you will need to change the the DNS settings by following the instructions [here](http://xslab.com/2013/08/how-to-change-dns-settings-on-android/). You also need to make sure the device and your local machine are using the same network.
 
-## Step 5: Install the CA file and the user certificate on the Android device
+## Step 4: Install the CA file and the user certificate on the Android device
 
 ### On an Android emulator
```

Comment:
Maybe the references here and below could be changed to `Mobile` now, instead of android?

Reply:
Yeah, I think I will add something about ios emulators here as well.